### DISCUSSION TITLE
WorldJam

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#2676] [General]
WorldJam should be promoted on Jamulus Web Page

https://worldjam.vip/homepage.php

**Comment by ann0see** [comment]
I think it already is on the Demos Page. But I think it was decided to remain more strictly devided. @pljones and @gilgongo should know.

**Reply by pljones** [reply]
Jamulus should be promoted on the WorldJam page.  But saying WorldJam should be **promoted** on the Jamulus page... you'd need to talk to all the other tech providers they use, like all the hosting providers, etc.  Jamulus is just part of their solution.

So no, I don't think we should specifically mention WorldJam over and above any other example use case of the software.

**Reply by rdica** [reply]
:+1:
